rollup/core/objects/RollupSettings__c/fields/IsEnabled__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>IsEnabled__c</fullName>
+    <defaultValue>false</defaultValue>
```

Comment:
From an admin perspective, I feel like it makes more sense for this to default to `true` instead. In my experience with DLRS (or other other options), I typically only want to disable rollups temporarily in a few scenarios:

- Data migration/data import - temporarily disable rollups at a user-level for a 'data migration' user. On projects where the migrated data is already fully transformed & ready to load, then there's no need for rollups to run during migration jobs.
- Data cleanup - temporarily disable rollups for admins at a profile-level during some kind of data cleanup effort.

Outside of those 2 scenarios, I generally would want rollups to be enabled for everyone else so that the data is always up to date (default == true).

Reply:
Works for me, and that's great feedback